src/state/categories/actions.js

```diff
@@ -81,12 +81,12 @@ export const fetchCategoryTree = (categoryCode = ROOT_CODE, recursive) => async
       const response = await fetchCategoryChildren(categoryCode)(dispatch);
       categoryTree = response.payload;
       if (recursive) {
-        // eslint-disable-next-line
-        categoryTree.map(cat => {
+        categoryTree.map((cat) => {
           if (cat.children && cat.children.length > 0) {
-            // eslint-disable-next-line
+            // eslint-disable-next-line no-use-before-define
             return dispatch(handleExpandCategory(cat.code, true));
```

Comment:
this should be fixed as the linter wants it to be fixed instead of disabling the rule